ironfish-rust-nodejs/index.d.ts

```diff
@@ -3,6 +3,12 @@
 
 /* auto-generated by NAPI-RS */
 
+export class ExternalObject<T> {
+  readonly '': {
+    readonly '': unique symbol
+    [K: symbol]: T
+  }
+}
```

Comment:
This keeps being generated, can we revert this change and ask the rust devs why this keeps being generated and if its ok to commit?

Reply:
https://iron-fish.slack.com/archives/C02PF56HESK/p1666109061641279